gulpfile.js

```diff
@@ -25,12 +27,45 @@ gulp.task('process-scss', () => gulp.src(['stylesheets/master.scss', 'stylesheet
 	.pipe(rename({ suffix: '.min' }))
 	.pipe(gulp.dest('dist/css')));
 
-gulp.task('minify-js', ['process-master-js', 'process-homepage-js', 'process-app-js', 'move-page-specific-js'], () => gulp.src(['dist/js/**/*.js'])
-	.pipe(uglify())
-	.on('error', (err) => { util.log(util.colors.red('[Error]'), err.toString()); })
-	.pipe(rename({ suffix: '.min' }))
-	.pipe(gulp.dest('dist/js')));
+gulp.task('minify-js', ['process-master-js', 'process-homepage-js', 'process-app-js', 'move-page-specific-js', 'process-featured-events-widget-js'], () => {
+	return gulp.src(['dist/js/**/*.js', '!dist/js/lib/*.js'])
+		.pipe(uglify())
+		.on('error', (err) => { util.log(util.colors.red('[Error]'), err.toString()); })
+		.pipe(rename({ suffix: '.min' }))
+		.pipe(gulp.dest('dist/js'));
+});
+
+
+gulp.task('download-angular-libs', () => {
+	const libFiles = [
```

Comment:
Downloading the files on every build doesn't make too much sense to me. Would you add them to a `/js/vendor/angular/` folder?